### PR TITLE
refactor: warning when the plugin disabled by app is enabled implicitly

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by popomore** [comment]
@gxcsoccer 看看，原来的报错改成 warn 了。

File: lib/loader/mixin/plugin.js [hunk 1]
```diff
@@ -109,20 +109,19 @@ module.exports = {
 
       // disable the plugin that not match the serverEnv
       if (env && plugin.env.length && plugin.env.indexOf(env) === -1) {
-        debug('Disable %j, as env is %j but got %j', name, plugin.env, env);
+        this.options.logger.info('Disable %j, as env is %j but got %j', name, plugin.env, env);
```
Comment (dead-horse) [inline lib/loader/mixin/plugin.js]: 这个 info 的信息不太容易看懂，改成 `plugin "xxx" disabled by env unmatch, plugin require env(test, prod), but application's env is local. `

Comment (popomore) [reply lib/loader/mixin/plugin.js]: 好的

**Comment by popomore** [comment]
@gxcsoccer 看看

**Comment by popomore** [comment]
没问题我合并了？

**Comment by atian25** [comment]
+1

**Pull request merged**